Makefile

```diff
@@ -61,4 +73,44 @@ clean:: # @HELP remove all the build artifacts
 	rm -rf ./build/_output ./vendor ./cmd/onos-config/onos-config ./cmd/onos/onos
 	go clean -testcache github.com/onosproject/onos-config/...
 
-
+local-helmit: # @HELP Copies a local version of the helmit dependency into the vendor directory
+ifdef LOCAL_HELMIT
+	rm -rf vendor/github.com/onosproject/helmit/go
+	mkdir -p vendor/github.com/onosproject/helmit/go
+	cp -r ${LOCAL_HELMIT}/go/* vendor/github.com/onosproject/helmit/go
+endif
+
+local-onos-api: # @HELP Copies a local version of the onos-api dependency into the vendor directory
+ifdef LOCAL_ONOS_API
+	rm -rf vendor/github.com/onosproject/onos-api/go
+	mkdir -p vendor/github.com/onosproject/onos-api/go
+	cp -r ${LOCAL_ONOS_API}/go/* vendor/github.com/onosproject/onos-api/go
+endif
+
+local-onos-lib-go: # @HELP Copies a local version of the onos-lib-go dependency into the vendor directory
+ifdef LOCAL_ONOS_LIB_GO
+	rm -rf vendor/github.com/onosproject/onos-lib-go
+	mkdir -p vendor/github.com/onosproject/onos-lib-go
+	cp -r ${LOCAL_ONOS_LIB_GO}/* vendor/github.com/onosproject/onos-lib-go
+endif
+
+local-onos-ric-sdk-go: # @HELP Copies a local version of the onos-ric-sdk-go dependency into the vendor directory
```

Comment:
I am not sure this is a good idea if we want to be consistent in all repos in this organization. The problem is different repos have different dep to different subsystems and each of those dep can be replaced. Previously, it was not needed to us any of these things to get local dep working.

Reply:
That's consistent with the way VOLTHA handles that: https://github.com/opencord/voltha-go/blob/master/Makefile#L70-L84

It's not required to have those "helpers" in the makefile, as long as the local `vendor` folder is mounted in the container the dependency replacement will work. I found that useful in the past so that the workflow looks like:

```
// make changes in the onos-api repo (wherever it is on the local fs)

// build the onos-config container wherever those changes are
LOCAL_ONOS_API=<path-to-onos-api-repo> make kind
```
If everything works commit the changes in the `onos-api` repo, no copy/paste needed from the `vendor` folder, once release update `go.mod` in `onos-config`, run `make mod-update` and commit.

Reply:
Good. If those helpers are optional and vendor folder is already taken care of that replace thing then it is great. I thought we just support those deps for local replace.

Reply:
Ah no, my bad for not explaining :)

You can replace anything you want in the `vendor` folder and it will work fine, these helpers are good for dependencies we manage as often we work on both in parallel, this way you don't have to copy them around manually (reducing the margin for errors).

If we adopt this approach on all the repos and you're testing a change that spans Config, Topo and CLI, you can simply run the same `LOCAL_ONOS_API=<path-to-onos-api-repo> make kind` in the 3 repos to have the image with your changes in `onos-api`